helm/templates/operator-deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
       containers:
       - image: {{ template "cloudwatch-agent-operator.image" . }}
         args:
+        - {{ printf "--auto-annotation-config=%s" (.Values.manager.autoAnnotation | toJson) | quote }}
```

Comment:
I spent a lot of time trying to understand why the following wouldnt work:
```
- "--auto-annotation-config={{ (.Values.manager.autoAnnotation | toJson | quote) }}"
```
It fails with an error saying `Error: YAML parse error on amazon-cloudwatch-observability/templates/operator-deployment.yaml: error converting YAML to JSON: yaml: line 29: did not find expected key`

After a lot of trial & error, the above is what finally worked to achieve the same end goal.

Reply:
Does `quote` add double-quotes which are interfering with the literal double quotes you have here?

Is there a way to have Helm spit out the raw text that it generates after processing?

Reply:
In the actual deployment manifest, it looks like follows:
```
- "--auto-annotation-config={\"java\":{\"daemonsets\":[],\"deployments\":[],\"namespaces\":[],\"statefulsets\":[]},\"python\":{\"daemonsets\":[],\"deployments\":[],\"namespaces\":[],\"statefulsets\":[]}}"
```